main.css

```diff
@@ -0,0 +1,180 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300..700&display=swap");
+
+*::before,
+*::after {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: Poppins, sans-serif;
+  display: grid;
+  place-items: center;
+  height: 100vh;
+  background: #e6e6e6;
+  width: 100%;
+}
+
+.container {
+  display: grid;
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 327px;
+  margin-block: 2rem;
+  gap: 2rem;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+  object-fit: cover;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  overflow: hidden;
+  box-shadow: 0 0.1rem 1rem rgba(0, 0, 0, 0.1);
+  border-radius: 1em;
+  background: #fff;
+}
+
```

Comment:
[ Required Changes ]

- [ ] Please add bottom spacing on the cards. 
Your Design
![image](https://user-images.githubusercontent.com/50941074/159133835-05e8e503-f8f5-4fb1-97e7-fb88cf9de82d.png)

The Figma Design
![image](https://user-images.githubusercontent.com/50941074/159133800-ffe6673a-0cd3-463b-a7e7-be229f305807.png)

Hint: Add this bottom margin style to fix it. 
`margin-bottom: 88px;`

Reply:
Done